helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1383,3 +1384,18 @@ operator:
 certmanager:
   enabled: false
   installCRDs: true
+
+
+################################################################################
+# Helm Chart Feature Gates.
+# The following feature gates are used to enable/disable features in the Helm chart
+# that are not yet ready for general availability.
+# Options in this section are not guaranteed to be stable and may change at any time.
+################################################################################
+
+featureGates:
+  # Use Light Prometheus Receiver for metrics collection from discovered Prometheus endpoints.
+  # https://github.com/signalfx/splunk-otel-collector/tree/main/internal/receiver/lightprometheusreceiver
+  # Light Prometheus Receiver is optimized for performance and reduced memory footprint.
+  # From the other hand, it does not support all Prometheus configuration options.
+  useLightPrometheusReceiver: false
```

Comment:
Note: I added support for using [feature gates per major component](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#using-feature-gates) a while ago (agent, clusterReceiver, gateway, etc). I'm fine with adding a root featureGates option, but it does seem like only the agent is using this at the moment.

Reply:
Not sure how useful they are. I'd recommend replacing them with more generic `extraCmdAgrs`. WDYT?

Reply:
> I'm fine with adding a root featureGates option, but it does seem like only the agent is using this at the moment.

To clarify, I'm adding _helm chart_ feature gates, not collector feature gates

Reply:
> I'd recommend replacing them with more generic extraCmdAgrs. WDYT?

This sounds like a useful change to me, I've wanted functionality like this a couple times.